Makefile

```diff
@@ -1,2 +1,38 @@
-shell:
+
+# Help message borrowed from https://github.com/edx/devstack, which borrowed it from https://github.com/pydanny/cookiecutter-djangopackage.
+help: ## display a help message
+	@echo $(MAKEFILE_LIST)
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[\.%a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+shell: ## run a shell on the studio-frontend container
 	docker exec -it dahlia.studio-frontend /bin/bash
+
+dev.up: ## bring up studio-frontend container
+	docker-compose up -d studio-frontend
+
+dev.down: ## stop and remove studio-frontend container
+	docker-compose down studio-frontend
+
+dev.stop: ## stop studio-frontend container but do not remove
+	docker-compose stop studio-frontend
+
+dev.start: ## start stopped studio-frontend container
+	docker-compose start studio-frontend
```

Comment:
How exactly do `up`, `build`, and `start` compare? I feel like we don't need all of them, but I'm not 100% on the difference. I usually only use up (which is similar to build+start?)

I'd also like to request a single command for "stop, remove, rebuild (without cache), and restart"

Reply:
They're not terribly different, and we could definitely roll them up into one if that's how we're most likely to use them. The way I understand it is:

- dev.up = using an existing image tagged edxops/studio-frontend (or pulling down from docker hub), start the container

- dev.start = restart an existing container that was brought up and then stopped, since dev.down actually removes the container; I guess this would save us the step of having to rerun the entrypoint command the way we would if we did dev.down and dev.up, but I'm not sure how big of a gain that is

- dev.build = thinking more about it, I don't think it's super useful, since we have the bind mounting. I thought maybe it would be useful to either re-build an image locally that differs from the one on DockerHub (with same tag) or to build an image with a different tag for testing purposes. Could there be local changes that don't affect the image until the image is rebuilt? For example, a change to the Dockerfile wouldn't affect the running container, and you'd want to test it out by rebuilding. The re-tagging would allow you to keep the existing image without overwriting it.

Does that help us determine what would be useful? I'm fine removing any that seem superfluous. 

And that's a good idea. I'll add some kind of start from scratch command.